Kickstarter-iOS/ViewModels/SettingsAccountViewModelTests.swift

```diff
@@ -33,7 +33,7 @@ internal final class SettingsAccountViewModelTests: TestCase {
 
   func testReloadData() {
     let userEnvelope = UserEnvelope(me: GraphUser.template)
-    let mockService = MockService(fetchGraphUserAccountFieldsResponse: userEnvelope)
+    let mockService = MockService(fetchGraphUserResponse: userEnvelope)
```

Comment:
I renamed the parameter to reflect the change in functionality so you'll see this in a handful of files.